docs/UserGuide.md

```diff
@@ -114,10 +114,18 @@ Cautions are placed in this guide to serve as warnings for certain actions.
 * Prefixes can be in any order.<br>
   e.g. If the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
+* If an input contains `"/"`, you cannot put spacing after its prefix.<br>
+  e.g. `s/Java/Python` will be interpreted as skills in `Java/Python` but if you specify
+  `n/Bob s/  Java/Python`, `Java/` will be interpreted as a prefix.
+
 * If an input is expected only once in the command but you specified it multiple times,
   only the last occurrence of the input will be taken.<br>
   e.g. If you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
+* If a valid prefix is given twice but you only specified input for the last occurrence of prefix, you will not receive errors.
```

Comment:
I feel like if we claim 'you will not receive errors', there might be some groups flagging this as a false claim.

```suggestion
* If a valid prefix is given twice but you only specified the input for the last occurrence of said prefix, then the input will be accepted.
```